### DISCUSSION TITLE
Why is buyer legal type rl-aut not in the list validate by the schematron?

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#357] [Q&A] [answered]
Why is buyer legal type rl-aut not in the list validate by the schematron?

rl-aut is in the codelist for buyer-legal-type.

  <Value ColumnRef="code">
            <SimpleValue>rl-aut</SimpleValue>
         </Value>
         <Value ColumnRef="Name">
            <SimpleValue>Regional or local authority</SimpleValue>
         </Value>

But why is rl-aut not included in the schematron validation? 

	<rule context="/*/cac:ContractingParty/cac:ContractingPartyType/cbc:PartyTypeCode[@listName='buyer-legal-type']">
		<assert id="BR-BT-00011-0052" role="ERROR" test="normalize-space(.) = ('body-pl', 'body-pl-cga', 'body-pl-la', 'body-pl-ra', 'cga', 'def-cont', 'eu-ins-bod-ag', 'grp-p-aut', 'int-org', 'la', 'org-sub', 'org-sub-cga', 'org-sub-la', 'org-sub-ra', 'pub-undert', 'pub-undert-cga', 'pub-undert-la', 'pub-undert-ra', 'ra', 'spec-rights-entity')">rule|text|BR-BT-00011-0052</assert>
	</rule>

**Comment by ferrakaop** [comment, marked as answer]
The field for buyer legal type is validated against the tailored codelist eforms-buyer-legal-type
https://docs.ted.europa.eu/eforms/latest/reference/code-lists/eforms-buyer-legal-type.html

This codelist doesn't contain the code rl-aut, which has been deprecated - but it's still available in the parent codelist buyer-legal-type, which isn't actually used for validation.
